mod_reforged/hooks/config/strings.nut

```diff
@@ -186,6 +186,21 @@ local vanillaDescriptions = [
  			}]
 	 	}),
 	},
+	{
+		ID = "perk.anticipation",
+		Key = "Anticipation",
+		Description = ::UPD.getDescription({
+			Fluff = "I saw these coming!",
+	 		Effects = [{
+ 				Type = ::UPD.EffectType.Passive,
+ 				Description = [
+					"Take reduced damage from the first " + ::MSU.Text.colorGreen(2) + " attacks you receive each battle.",
```

Comment:
These both points should be merged into a single point that reads much better: `Gain percentage damage reduction equal to your Ranged Defense for the first two attacks, ranged or melee, against you in every battle.`

Reply:
I prefer the more verbose way of describing the effect, especially since this perk description has the space to spare for it.
Both of my bullet points convey their own important effect and this separation makes them more readable.
I view this way of formatting as an exception that we can sometimes employ if we see it fit.

Reply:
Yeah because we're keeping the 10% tile per tile part as well, it's better to split it for now. But we have to say that the reduction is a percentage. The current statement is inaccurate i.e. it says the reduction is `equal to your Ranged Defense + an additional 10%...`. Whereas it should say the reduction is `equal to a percentage equal to your current Ranged Defense + an additional 10%...`. 

Please replace `+` with `and` or `plus`. It's cleaner to keep tooltips free of mathematical symbols and that's in line with the other tooltips too.